tests/acceptance/01_vars/02_functions/cf_version.cf

```diff
@@ -0,0 +1,55 @@
+###########################################################
+#
+# Test cf_version convenience functions
+#
+###########################################################
+
+body common control
+{
+    inputs => { "../../default.cf.sub" };
+    bundlesequence => { default($(this.promise_filename)) };
+    version => "1.0";
+}
+
+###########################################################
+
+bundle agent test
+{
+  vars:
+    "after_newer" string => "yes",
+      if => cf_version_after("1.0.0");
```

Comment:
Could add some test cases where you don't specify patch version.

`if => cf_version_after("3.12")`

`if => cf_version_at("3")`

Reply:
Regarding the second one: would be nice to make sure that the tests work between major functions though, right?

Reply:
Could be, but not a big deal, imo.